package.json

```diff
@@ -42,7 +45,8 @@
     "ganache": "sh tools/runGanacheCli.sh",
     "build-package": "node tools/build_package.js",
     "prepare": "npm run compile-all && npm run build-package",
-    "deploy:gateway": "npm run compile && node tools/blue_deployment/scripts/step1_origin_contracts.js"
+    "deploy:gateway": "npm run compile && node tools/blue_deployment/scripts/step1_origin_contracts.js",
+    "lint": "eslint {migrations,test,test_integration,tools} -c .eslintrc.json --ext js"
```

Comment:
maybe add a pre-commit hook to run this.  Refer [this.](https://github.com/OpenSTFoundation/mosaic.js/blob/bcb49479763f782b35cf447c4fc893e16a1a6384/package.json#L63)

Reply:
It still returns more than 700 issues. Not sure a pre-commit hook helps 🤔